src/common/utils/gather-tokens.util.ts

```diff
@@ -1,17 +1,27 @@
 import { UserEntity } from "@entities/index";
 import { GuildMember, User } from "discord.js";
-import { convertMilisToString } from "./converter.util";
+import { getDuration } from "./get-duration.util";
+import { CustomClient } from "@client/custom-client";
 
 interface TokensProps {
   availableTokens: string[];
   tokens: { [k: string]: string | number };
 }
 
 export function gatherProfileTokens(
+  client: CustomClient,
```

Comment:
No need to pass client, you can just pass usersInVoice collection here

Reply:
Will agree with this one.